source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Summary
+
+Skydive is an open source real-time network topology and protocols analyzer providing a comprehensive way of understanding what is happening in your network infrastructure.
+The common use cases will be, troubleshooting, monitoring, SDN integration and much more.
+It has features such as:
+* Topology capturing - Captures network topology, interface, bridge and more
+* Flow capture - Distributed probe, L2-L4 classifier, GRE, VXLAN, GENEVE, MPLS/GRE, MPLS/UDP tunnelling support
+* Extendable - Support for external SDN Controllers or container based infrastructure, OpenStack, OpenContrail, Docker. Supports extensions through API
+* Distributed
+
+[Skydive network](http://skydive.network/) - Link to Skydive site
+
+## Installation steps
+1. git clone https://github.com/skydive-project/skydive.git
+2. Create inventory file
+
+```ini  
+[skydive:children]
+analyzers
+agents
+
+[skydive:vars]
+skydive_listen_ip=0.0.0.0
+skydive_fabric_default_interface=ovirtmgmt
+
+skydive_os_auth_url=https://<FQDN>:35357/v2.0
+skydive_os_service_username=<username>
+skydive_os_service_password=<password>
+skydive_os_service_tenant_name=service
+skydive_os_service_domain_name=Default
+skydive_os_service_region_name=RegionOne
+
+[analyzers]
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
```

Comment:
it might be helpful if the password of the host has not the same representation like the oVirt password, e.g.
`ansible_ssh_pass=<ssh_password>`

Reply:
Done